collect_app/src/main/java/org/odk/collect/android/utilities/DateTimeUtils.java

```diff
@@ -125,4 +125,29 @@ public static DatePickerDetails getDatePickerDetails(String appearance) {
 
         return new DatePickerDetails(datePickerType, datePickerMode);
     }
+
+    /**
+     * Function to convert milliseconds time to Hours:Minutes:Seconds Format
+     */
+    public static String millisToTimer(long milliseconds) {
```

Comment:
Did you consider Joda-Time, or whatever the modern equivalent might be?

Reply:
No, I don't know about that. Can you please guide me how to do that?

Reply:
Something like—or perhaps exactly—this:
```
    public static String millisToTimer(long milliseconds) {
        DateTimeFormatter fmt = DateTimeFormat.shortTime();
        return fmt.print(new DateTime(milliseconds));
    }
```